File: code/c/hello_world.c
/*
 * TEKNOFEST C Örneği - Hello World
 * Embedded sistemler ve düşük seviye programlama için
 */

#include <stdio.h>

int main() {
    printf("🚀 TEKNOFEST'e Hoş Geldiniz!\n");
    printf("C ile embedded sistemler ve performans kritik uygulamalar geliştirebilirsiniz.\n");
    
    // Basit hesaplama
    int takim_sayisi = 10;
    int yarisma_sayisi = 5;
    int toplam_katilim = takim_sayisi * yarisma_sayisi;
    
    printf("\n📊 İstatistikler:\n");
    printf("Takım Sayısı: %d\n", takim_sayisi);
    printf("Yarışma Sayısı: %d\n", yarisma_sayisi);
    printf("Toplam Katılım: %d\n", toplam_katilim);
    
    return 0;
}

File: code/go/hello_world.go
package main

/*
 * TEKNOFEST Go Örneği - Hello World
 * Concurrent ve scalable uygulamalar için
 */

import "fmt"

func main() {
	fmt.Println("🚀 TEKNOFEST'e Hoş Geldiniz!")
	fmt.Println("Go ile concurrent ve scalable sistemler geliştirebilirsiniz.")
	
	// Basit hesaplama
	takimSayisi := 10
	yarismaSayisi := 5
	toplamKatilim := takimSayisi * yarismaSayisi
	
	fmt.Println("\n📊 İstatistikler:")
	fmt.Printf("Takım Sayısı: %d\n", takimSayisi)
	fmt.Printf("Yarışma Sayısı: %d\n", yarismaSayisi)
	fmt.Printf("Toplam Katılım: %d\n", toplamKatilim)
}

